### ISSUE TITLE
Add option to zoom the image in the gallery component

### ISSUE THREAD

**Issue by dreamfarer** [sopra-fs25-group-20/sopra-fs25-group-20-client#86]

**Comment by dreamfarer** [comment]
This optional requirement cannot be implemented due to time constraints. Previously unplanned usability and accessibility features take priority in order to provide a polished experience.
